src/main/java/seedu/address/commons/util/DateTimeUtil.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.commons.util;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Utilities about date and time
+ */
+public class DateTimeUtil {
+    /**
+     * Parse string into LocalDateTime
+     * @param str the string to be parsed
+     * @return The parsed LocalDateTime
+     * @throws DateTimeParseException If the string cannot be parsed into LocalDateTime
+     */
+    public static LocalDateTime parseString(String str) throws DateTimeParseException {
+        DateTimeFormatter formatter1 = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm[:ss]");
+        LocalDateTime result = null;
+        try {
+            result = LocalDateTime.parse(str, formatter1);
+        } catch (DateTimeParseException e2) {
+            LocalDateTime now = LocalDateTime.now();
+            String appendDate = now.getYear() + "-" + now.getMonthValue() + "-" + now.getDayOfMonth() + " ";
+            result = LocalDateTime.parse(appendDate + str, formatter1);
+        }
```

Comment:
I think should be more reasonable to take care of the case where the user only puts in date (since this is a more general information) instead of the case where the user only puts in time. 

In case you wanna handle both, can you find a way to mkae sure that the code won't be too nested?

Reply:
You can resolve this one without changing the code, but for the other conversations, can you please make the change? Thanks!